docs/docs/how-tos/get-started/1-build-basic-chatbot.md

```diff
@@ -0,0 +1,208 @@
+# Build a basic chatbot
+
+In this tutorial, you will build a basic chatbot. This chatbot is the basis for the following series of tutorials where you will progressively add more sophisticated capabilities, and be introduced to key LangGraph concepts along the way. Let’s dive in! 🌟
+
+## Prerequisites
+
+Before you start this tutorial, ensure you have the following:
+
+- An API key for an LLM. This tutorial series uses Anthropic.
+
+## 1. Install packages
```

Comment:
numbering is off

<img width="283" alt="image" src="https://github.com/user-attachments/assets/bf3f9995-9581-458b-a52c-6c1e619908ed" />